website/static/js/osfHelpers.js

```diff
@@ -361,7 +361,12 @@ var mapByProperty = function(list, attr) {
     });
 };
 
-
+/** 
+  * Replaces encoded <, >, and & with decoded values
+  */
+var decodeText = function(text){
+    return text.replace(/&lt;/g, '<').replace(/&gt;/g, '>').replace(/&amp;/g, '&');
```

Comment:
A note:

In the past, we very intentionally did not decode `<` and `>` on the front end. I'd be happy to walk you through the logic for that in person, but suffice to say it involves an edge case of bleach. Has the rationale for that design decision changed?